pkg/vcs/vcs.go

```diff
@@ -8,13 +8,62 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"net/mail"
 	"regexp"
+	"sort"
 	"strings"
 	"time"
 
 	"github.com/google/syzkaller/pkg/osutil"
 )
 
+type RecipientType int
+
+const (
+	OpenList RecipientType = iota
+	Reviewer
+	FileList
+	Maintainer
+	Supporter
+)
+
+func (t RecipientType) String() string {
+	return [...]string{"open list", "reviewer", "file list", "maintainer", "supporter"}[t]
+}
+
+type RecipientInfo struct {
+	Address mail.Address
+	Type    RecipientType
+}
+
+type Recipients []RecipientInfo
+
+func (r Recipients) GetEmails(to bool) []string {
+	emails := []string{}
+	for _, user := range r {
+		if to && user.Type > 1 {
+			emails = append(emails, user.Address.Address)
+		} else if !to && user.Type < 2 {
```

Comment:
Can't this be replaced with just `else`? I would assume we want to put every recipient either to To or to Cc.